src/daft-plan/src/builder.rs

```diff
@@ -658,6 +683,7 @@ impl ParquetScanBuilder {
             ))),
             self.infer_schema,
             self.schema,
+            None,
```

Comment:
I think here we should either add a TODO or add `file_path_column` to the `ParquetScanBuilder`, then add the argument to the SQL side as well in `read_parquet.rs`.